src/client/src/routes/CreateUserPage/components/CreateUser.js

```diff
@@ -6,6 +6,7 @@ function CreateUser(props) {
   const [username, setUsername] = useState("");
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const [response, setResponse] = useState("");
```

Comment:
maybe a better option would be to combine all the states in a single object and just update that. would also reduce the length of the code ( [values, setValues] = useState({ username: ' ', email: ' ', ... }) )

Reply:
Wouldn't it make the code hard to understand?

Reply:
both options are valid... I don't find either of them superior on the other. It can be changed later if some specific reason will make such decision significant